modules/docs/arrow-docs/docs/docs/quickstart/libraries/README.md

```diff
@@ -10,178 +10,246 @@ Arrow is a modular set of libraries that build on top of each other to provide i
+Recursion schemes is a construct to work with recursive data structures in a way that decouples structure and data, and allows for ergonomy and performance improvements.
 
-Typeclasses: [`Fx`]({{ '/docs/effects/fx/' | relative_url }})
+Dependency: `arrow-recursion-data`
 
-### arrow-recursion
+### arrow-recursion-data
 
-{:.advanced}
-advanced
+```groovy
+dependencies {
+    compile "io.arrow-kt:arrow-recursion-data:$arrow_version"
+}
+```
 
-Recursion schemes is a construct to work with recursive data structures in a way that decuples structure and data, and allows for ergonomy and performance improvements.
+This library includes the datatypes and typeclasses in Recursion schemes. 
 
 Datatypes: [`Fix`]({{ '/docs/recursion/fix/' | relative_url }}), [`Mu`]({{ '/docs/recursion/mu/' | relative_url }}), [`Nu`]({{ '/docs/recursion/nu/' | relative_url }})
 
 Typeclasses: [`Corecursive`]({{ '/docs/recursion/corecursive/' | relative_url }}), [`Recursive`]({{ '/docs/recursion/recursive/' | relative_url }}), [`Birecursive`]({{ '/docs/recursion/birecursive/' | relative_url }})
 
+Dependency: `arrow-core`
+
 ### arrow-integration-retrofit-adapter
 
-{:.advanced}
-advanced
+```groovy
+dependencies {
+    compile "io.arrow-kt:arrow-integration-retrofit-adapter:$arrow_version"
+}
+```
 
 The [adapter]({{ 'docs/integrations/retrofit/' | relative_url }}) is a library that adds integration with Retrofit, providing extensions functions and/or classes to work with Retrofit by encapsulating the responses in the chosen datatypes, through the use of typeclasses.
 
+Dependency: `arrow-fx`
+
 ### arrow-free
 
-{:.advanced}
-advanced
+```groovy
+dependencies {
+    compile "io.arrow-kt:arrow-free:$arrow_version"
+}
+```
 
 The [Free datatype]({{ '/docs/free/free/' | relative_url }}) is a way of interpreting domain specific languages from inside your program, including a configurable runner and flexible algebras.
 This allows optimization of operations like operator fusion or parallelism, while remaining on your business domain.
 
+Dependency: `arrow-free-data`
+
+### arrow-free-data
+
+```groovy
+dependencies {
+    compile "io.arrow-kt:arrow-free-data:$arrow_version"
+}
+```
+
+This library includes the datatypes in Free. 
+
 Datatypes: [`Free`]({{ '/docs/free/free/' | relative_url }}), [`FreeApplicative`]({{ '/docs/free/freeapplicative/' | relative_url }}), [`Cofree`]({{ '/docs/free/cofree/' | relative_url }}), [`Yoneda`]({{ '/docs/free/yoneda/' | relative_url }}), [`Coyoneda`]({{ '/docs/free/coyoneda/' | relative_url }})
 
+Dependency: `arrow-core`
+
 ### arrow-aql
 
-{:.intermediate}
-intermediate
+```groovy
+dependencies {
+    compile "io.arrow-kt:arrow-query-language:$arrow_version"
+}
+```
 
 This [Arrow Query Library]({{ '/docs/aql/intro/' | relative_url }}) focuses on bringing SQL-like syntax to Arrow datatypes.
 
+Dependency: `arrow-core`
+
 ## Annotation processors
```

Comment:
I think this is awfully low on the page given that a lot of people ask for kapt setup, no? Should we perhaps add a small table of contents at the top so people can jump here?

Reply:
The more important thing is the core libraries everything else is still experimental and with the compiler plugins, some old libraries like arrow-syntax may as well disappear.
I only want that to be our focus even though we have other awesome projects.